### ISSUE TITLE
TensorBoard not working with 0.11 RC1

### ISSUE THREAD

**Issue by jkschin** [tensorflow/tensorflow#5341]
All I see is a blank page when I go to localhost:6006

Browser: Chrome (Version 54.0.2840.71 (64-bit))
Here's the full stack trace:

127.0.0.1 - - [02/Nov/2016 08:59:41] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /webcomponentsjs/webcomponents-lite.min.js HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /dist/bazel-html-imports.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /lib/css/global.css HTTP/1.1" 200 -
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /dist/tf-tensorboard.html HTTP/1.1" 200 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /polymer/polymer.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /iron-icons/iron-icons.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-tabs/paper-tabs.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-dialog/paper-dialog.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-toolbar/paper-toolbar.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-button/paper-button.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-checkbox/paper-checkbox.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-header-panel/paper-header-panel.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /lodash/lodash.min.js HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-slider/paper-slider.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-input/paper-input.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /d3/d3.js HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-styles/paper-styles.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-dropdown-menu/paper-dropdown-menu.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-menu/paper-menu.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-item/paper-item.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /iron-collapse/iron-collapse.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /plottable/plottable.js HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /plottable/plottable.css HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-icon-button/paper-icon-button.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-toggle-button/paper-toggle-button.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /lodash/lodash.min.js HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /graphlib/dist/graphlib.core.js HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /dagre/dist/dagre.core.js HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /lodash/lodash.min.js HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /graphlib/dist/graphlib.core.js HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /iron-flex-layout/iron-flex-layout.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /iron-list/iron-list.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-item/all-imports.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-progress/paper-progress.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-radio-group/paper-radio-group.html HTTP/1.1" 404 -
127.0.0.1 - - [02/Nov/2016 08:59:41] code 404, message Not Found
127.0.0.1 - - [02/Nov/2016 08:59:41] "GET /paper-tooltip/paper-tooltip.html HTTP/1.1" 404 -

**Comment by jkschin** [comment]
Upgraded to 0.11 RC2 and I still see the same errors.

**Comment by aselle** [comment]
@dsmilkov  could you please take a look at this issue?

**Comment by danmane** [comment]
I tried to reproduce this and couldn't. I tried the following things:
Linux - Build the pip package against master (not 0.11 rc2 due to bazel version compatibility issues), install resultant pip package, and launch TensorBoard. No problem.
Mac - Build the pip package against 0.11 rc2, install resultant pip package, and launch TensorBoard. No problem.
Mac - Bazel build tensorboard, and launch tensorboard from bazel-bin. No problem.

@jkschin Please provide a more detailed reproduction.

**Comment by jkschin** [comment]
@danmane 

```
git checkout master (commit number a50743836)
bazel build -c opt --config=cuda //tensorflow/tools/pip_package:build_pip_package
bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
sudo pip uninstall tensorflow
sudo pip install /tmp/tensorflow_pkg/tensorflow-0.11.0rc2-cp27-none-linux_x86_64.whl
python
import tensorflow as tf
tf.__version__ == '0.11.0rc2'
```

Bazel details
```
Build label: 0.3.2
Build target: bazel-out/local-fastbuild/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Fri Oct 7 17:25:10 2016 (1475861110)
Build timestamp: 1475861110
Build timestamp as int: 1475861110
```

Ran 
```
python cifar10_train.py
tensorboard --logdir=/tmp/cifar10_train
```

Got this still
```
127.0.0.1 - - [04/Nov/2016 13:49:15] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [04/Nov/2016 13:49:15] code 404, message Not Found
127.0.0.1 - - [04/Nov/2016 13:49:15] "GET /webcomponentsjs/webcomponents-lite.min.js HTTP/1.1" 404 -
127.0.0.1 - - [04/Nov/2016 13:49:15] "GET /dist/bazel-html-imports.html HTTP/1.1" 200 -
127.0.0.1 - - [04/Nov/2016 13:49:15] "GET /lib/css/global.css HTTP/1.1" 200 -
127.0.0.1 - - [04/Nov/2016 13:49:15] "GET /dist/tf-tensorboard.html HTTP/1.1" 200 -
127.0.0.1 - - [04/Nov/2016 13:49:15] code 404, message Not Found
127.0.0.1 - - [04/Nov/2016 13:49:15] "GET /paper-tabs/paper-tabs.html HTTP/1.1" 404 -
127.0.0.1 - - [04/Nov/2016 13:49:15] code 404, message Not Found
127.0.0.1 - - [04/Nov/2016 13:49:15] "GET /polymer/polymer.html HTTP/1.1" 404 -
127.0.0.1 - - [04/Nov/2016 13:49:15] code 404, message Not Found
127.0.0.1 - - [04/Nov/2016 13:49:15] code 404, message Not Found
127.0.0.1 - - [04/Nov/2016 13:49:15] "GET /iron-icons/iron-icons.html HTTP/1.1" 404 -
127.0.0.1 - - [04/Nov/2016 13:49:15] "GET /numericjs_numeric_min_js/file/numeric.min.js HTTP/1.1" 404 
```

**Comment by scanyameres** [comment]
Hi @jkschin,

I had the very same problem and my colleague just realised that we are calling tensorboard in different ways, and it was automatically solved. Instead of running tensorboard by

`tensorboard --logdir=path/to/log`

try running the tensorboard.py file in the tensorflow library. In my case:

`cd /usr/local/lib/python2.7/dist-packages/tensorflow/tensorboard
python tensorboard.py --logdir=path/to/log`

I hope it works for you as well!

**Comment by devashishshankar** [comment]
Had the same issue with both 0.11 and 0.11 RC2. @scanyameres's workaround worked for me.

**Comment by MatthiasWinkelmann** [comment]
I ran into the same problem – using Chrome. It's not browser-specific.

I had installed tf using the instructions at [https://www.tensorflow.org/versions/r0.11/get_started/os_setup.html#using-pip](https://www.tensorflow.org/versions/r0.11/get_started/os_setup.html#using-pip), specifically:

```
# Mac OS X, GPU enabled, Python 3.4 or 3.5:
(tensorflow)$ export TF_BINARY_URL=https://storage.googleapis.com/tensorflow/mac/gpu/tensorflow-0.11.0-py3-none-any.whl
```

The file ```dist/bazel-html-imports.html``` was not in ```/usr/local/lib/python3.5/site-packages/tensorflow/tensorboard/dist```. I copied a few files from github (```package.json```, ```bower.json```, etc.) and ran bower and npm install and somehow got it working – numericjs was also missing.

I can't make sense of the build system fast enough to contribute a useful pull request, but from browsing the changes, I'd guess @dsmilkov may be able to help.

The workaround by @scanyameres may work if there are two installations of tf – a default one using python3.x and the python2.7 version without this problem.

**Comment by jart** [comment]
Most of our TensorBoard build system stuff is going to be replaced in the next couple of weeks. I would recommend against expending too much mental effort into trying to figure it out or contributing until then. The new build is going to be amazing and beautiful. You're going to love it.

**Comment by tomzx** [comment]
I'm on Windows, v0.12.0.rc0 GPU (tensorflow_gpu-0.12.0rc0-cp35-cp35m-win_amd64.whl) and I face the same issue.

**Comment by Carmezim** [comment]
@tomzx at this moment only the [nightly builds](http://ci.tensorflow.org/view/Nightly/job/nightly-win/) have the fix for this issue, which will also be in the `0.12.0rc-1`. The other alternative is building it yourself.

**Comment by Carmezim** [comment]
Could this be closed as the fix is in and will be in the next imminent release?

**Comment by gunan** [comment]
Judging from the last comments, this should be resolved in 0.12.1
